domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/induction/InductionSchedule.kt

```diff
@@ -71,8 +71,4 @@ enum class InductionScheduleStatus(
   fun isExemptionOrExclusion(): Boolean {
     return isExemption || isExclusion
   }
-
-  fun includeExceptionOnSummary(): Boolean {
```

Comment:
Not strictly part of the bug fix, but as part of diagnosing it I ended up refactoring elements of `SessionSummaryService` and I've implemented this method as a private extension function in `SessionSummaryService` because that is closer and more relevant to where it is used